### ISSUE TITLE
Have error when develop

### ISSUE THREAD

**Issue by lchankiet** [xuopled/gatsby-source-google-docs#31]
Error: source-google-docs: Cannot read property 'generateAuthUrl' of undefined
![image](https://user-images.githubusercontent.com/40896000/64470041-a4026300-d166-11e9-839b-abe7be012939.png)

**Comment by vitostan** [comment]
This function `generateAuthUrl` was in the sample node.js code in the second step of the gatsby-source-google-docs: [Turn ON the Google Drive API](https://developers.google.com/drive/api/v3/quickstart/nodejs). 

So have you checked that the credential file name you have is same as the name in that piece of code (by default it's `credentials.json`)? 

I met the same error and solved by checking the thing above.

**Comment by xuopled** [comment]
@lchankiet is your issue resolved?

**Comment by voskresla** [comment]
I have the same issue.

**Comment by voskresla** [comment]
Okay, i found the problem.

Quick fix: in auth.js in getAuth() i change this
`const token = await getToken(tokenProps)`
to 
`const token = await getToken({...tokenProps, client})`

i think there is a problem in some transpailing process when gatsby build develop server. Because in original auth.js we have spread {...tokenProps} but in debug i see 
```
async function getAuth(_ref) {
  let {client_id, client_secret, redirect_uris} = _ref,
    tokenProps = _objectWithoutProperties(_ref, [
      "client_id",
      "client_secret",
      "redirect_uris",
    ])
```

**Comment by xuopled** [comment]
Published in `v1.14.2`
